buildspec.yml

```diff
@@ -23,9 +23,10 @@ phases:
 
   build:
     on-failure: ABORT
-    commands: 
+    commands:
       - | 
         docker build \
+          --cache-from $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME:$DEPLOYMENT_ENV-latest \
           -t $APP_NAME:$DEPLOYMENT_ENV-`cat commit_hash` \
           -f $DOCKERFILE_SERVER_PATH .
```

Comment:
One thing I think we'll need to add to this file is the build and push process for `Dockerfile.rake`.

I created a special image for our ECS scheduled task. That will be the image we apply to `KlaxonScheduledTaskDefinition`. 

So we'll need to add variable to the env. 

```
DOCKERFILE_RAKE_PATH: ./Dockerfile.rake
```

Then we need to add a second `docker build` process:
```
- |
   docker build --cache-from $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME-rake:$DEPLOYMENT_ENV-latest -t $APP_NAME-rake:$DEPLOYMENT_ENV-`cat commit_hash` -f $DOCKERFILE_RAKE_PATH
```
_note how we are building a new image with a special name signifying that it's the rake image: `$APP_NAME-rake:$DEPLOYMENT_ENV-`cat commit_hash`_

Then _I think_ we need to tag and push the new image up to ECR:
```
     docker tag $APP_NAME-rake:$DEPLOYMENT_ENV-`cat commit_hash` $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME-rake:$DEPLOYMENT_ENV-$VERSION
docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$APP_NAME-rake:$DEPLOYMENT_ENV-$VERSION
```

I am basing this information off of the [documents api buildspec](https://github.com/WPMedia/newsroom-documents-api/blob/develop/buildspec.yml), where we build and push multiple docker images. Feel free to recommend any name changes for clarity!